airflow/providers/apache/hive/hooks/hive.py

```diff
@@ -775,6 +775,23 @@ def table_exists(self, table_name, db='default'):
         except Exception:  # pylint: disable=broad-except
             return False
 
+    def drop_partitions(self, table_name, part_vals, delete_data=False, db='default'):
+        """
+        Drop partitions matching param_names input
+        >>> hh = HiveMetastoreHook()
+        >>> hh.drop_partitions(db='airflow', table_name='static_babynames',
+        part_vals="['2020-05-01']")
+        True
```

Comment:
Please add arguments information using `:param xxx:` and `:type xxx:`

Reply:
Done!